src/lib.rs

```diff
@@ -6,114 +6,325 @@
-    0b0010_1100, 0b1010_1100, 0b0110_1100, 0b1110_1100,
-    0b0001_1100, 0b1001_1100, 0b0101_1100, 0b1101_1100,
-    0b0011_1100, 0b1011_1100, 0b0111_1100, 0b1111_1100,
-
- 
-    0b0000_0010, 0b1000_0010, 0b0100_0010, 0b1100_0010,
-    0b0010_0010, 0b1010_0010, 0b0110_0010, 0b1110_0010,
-    0b0001_0010, 0b1001_0010, 0b0101_0010, 0b1101_0010,
-    0b0011_0010, 0b1011_0010, 0b0111_0010, 0b1111_0010,
-
-    0b0000_1010, 0b1000_1010, 0b0100_1010, 0b1100_1010,
-    0b0010_1010, 0b1010_1010, 0b0110_1010, 0b1110_1010,
-    0b0001_1010, 0b1001_1010, 0b0101_1010, 0b1101_1010,
-    0b0011_1010, 0b1011_1010, 0b0111_1010, 0b1111_1010,
-
-    0b0000_0110, 0b1000_0110, 0b0100_0110, 0b1100_0110,
-    0b0010_0110, 0b1010_0110, 0b0110_0110, 0b1110_0110,
-    0b0001_0110, 0b1001_0110, 0b0101_0110, 0b1101_0110,
-    0b0011_0110, 0b1011_0110, 0b0111_0110, 0b1111_0110,
-
-    0b0000_1110, 0b1000_1110, 0b0100_1110, 0b1100_1110,
-    0b0010_1110, 0b1010_1110, 0b0110_1110, 0b1110_1110,
-    0b0001_1110, 0b1001_1110, 0b0101_1110, 0b1101_1110,
-    0b0011_1110, 0b1011_1110, 0b0111_1110, 0b1111_1110,
-
- 
-    0b0000_0001, 0b1000_0001, 0b0100_0001, 0b1100_0001,
-    0b0010_0001, 0b1010_0001, 0b0110_0001, 0b1110_0001,
-    0b0001_0001, 0b1001_0001, 0b0101_0001, 0b1101_0001,
-    0b0011_0001, 0b1011_0001, 0b0111_0001, 0b1111_0001,
-
-    0b0000_1001, 0b1000_1001, 0b0100_1001, 0b1100_1001,
-    0b0010_1001, 0b1010_1001, 0b0110_1001, 0b1110_1001,
-    0b0001_1001, 0b1001_1001, 0b0101_1001, 0b1101_1001,
-    0b0011_1001, 0b1011_1001, 0b0111_1001, 0b1111_1001,
-
-    0b0000_0101, 0b1000_0101, 0b0100_0101, 0b1100_0101,
-    0b0010_0101, 0b1010_0101, 0b0110_0101, 0b1110_0101,
-    0b0001_0101, 0b1001_0101, 0b0101_0101, 0b1101_0101,
-    0b0011_0101, 0b1011_0101, 0b0111_0101, 0b1111_0101,
-
-    0b0000_1101, 0b1000_1101, 0b0100_1101, 0b1100_1101,
-    0b0010_1101, 0b1010_1101, 0b0110_1101, 0b1110_1101,
-    0b0001_1101, 0b1001_1101, 0b0101_1101, 0b1101_1101,
-    0b0011_1101, 0b1011_1101, 0b0111_1101, 0b1111_1101,
-
- 
-    0b0000_0011, 0b1000_0011, 0b0100_0011, 0b1100_0011,
-    0b0010_0011, 0b1010_0011, 0b0110_0011, 0b1110_0011,
-    0b0001_0011, 0b1001_0011, 0b0101_0011, 0b1101_0011,
-    0b0011_0011, 0b1011_0011, 0b0111_0011, 0b1111_0011,
-
-    0b0000_1011, 0b1000_1011, 0b0100_1011, 0b1100_1011,
-    0b0010_1011, 0b1010_1011, 0b0110_1011, 0b1110_1011,
-    0b0001_1011, 0b1001_1011, 0b0101_1011, 0b1101_1011,
-    0b0011_1011, 0b1011_1011, 0b0111_1011, 0b1111_1011,
-
-    0b0000_0111, 0b1000_0111, 0b0100_0111, 0b1100_0111,
-    0b0010_0111, 0b1010_0111, 0b0110_0111, 0b1110_0111,
-    0b0001_0111, 0b1001_0111, 0b0101_0111, 0b1101_0111,
-    0b0011_0111, 0b1011_0111, 0b0111_0111, 0b1111_0111,
-
-    0b0000_1111, 0b1000_1111, 0b0100_1111, 0b1100_1111,
-    0b0010_1111, 0b1010_1111, 0b0110_1111, 0b1110_1111,
-    0b0001_1111, 0b1001_1111, 0b0101_1111, 0b1101_1111,
-    0b0011_1111, 0b1011_1111, 0b0111_1111, 0b1111_1111
-];
+static BIT_REV_U8: [u8; 256] = [0b0000_0000,
```

Comment:
Please do not change formatting where manual arrangement was used for semantic purposes (this is a lookup table, not a list).